sites/site-utilities/statics/server/server.js

```diff
@@ -7,13 +7,23 @@ var path = require("path");
 var app = express();
 
 // Security enhancements provided by https://github.com/helmetjs/helmet
-app.use(helmet());
+// contentSecurityPolicy() omitted because it breaks modern implementation methods
+app.use(helmet.dnsPrefetchControl());
+app.use(helmet.expectCt());
+app.use(helmet.frameguard());
+app.use(helmet.hidePoweredBy());
+app.use(helmet.hsts());
+app.use(helmet.ieNoOpen());
+app.use(helmet.noSniff());
+app.use(helmet.permittedCrossDomainPolicies());
+app.use(helmet.referrerPolicy());
+app.use(helmet.xssFilter());
```

Comment:
Would love to do a follow-up to see if we can do this more efficiently.

Reply:
good suggestion.